rust/gui-client/src-tauri/tauri.conf.json

```diff
@@ -9,18 +9,35 @@
     "shortDescription": "Firezone",
     "linux": {
       "deb": {
+        "preInstallScript": "./linux_package/preinst",
+        "postInstallScript": "./linux_package/postinst",
+        "preRemoveScript": "./linux_package/prerm",
         "files": {
-          "/usr/lib/systemd/system/firezone-client-tunnel.service": "./deb_files/firezone-client-tunnel.service",
-          "/usr/lib/sysusers.d/firezone-client-tunnel.conf": "./deb_files/sysusers.conf"
+          "/usr/lib/systemd/system/firezone-client-tunnel.service": "./linux_package/firezone-client-tunnel.service",
+          "/usr/lib/sysusers.d/firezone-client-tunnel.conf": "./linux_package/sysusers.conf",
+          "/usr/bin/firezone-client-tunnel": "../../target/release/firezone-client-tunnel"
```

Comment:
The tunnel service binary is now embedded as a regular file. This means the bundling will only work for release builds but we never make bundles of the debug build anyway.

Reply:
👍